cfgov/core/views.py

```diff
@@ -123,3 +126,21 @@ def submit_comment(data):
                              })
 
     return response
+
+
+@csrf_exempt
+def csp_violation_report(request):
+    import pdb;pdb.set_trace
+    if request.method == 'POST':
+        try:
+            csp_dict = json.loads(request.body)['csp-report']
+        # bare except is non-ideal, but if parsing fails for any reason
+        # we need to abort
+        except:
+            return HttpResponseForbidden()
+
+        message_template = '{blocked-uri} blocked on {document-uri}, violated {violated-directive}'
```

Comment:
I'm looking for the "official" spec, but here's a description:

https://developer.mozilla.org/en-US/docs/Web/Security/CSP/Using_CSP_violation_reports